.github/workflows/gradle-check.yml

```diff
@@ -30,11 +30,14 @@ jobs:
       - name: Setup environment variables (PR)
         if: github.event_name == 'pull_request_target'
         run: |
+          echo "branch_name=$(jq --raw-output .pull_request.base.ref $GITHUB_EVENT_PATH)" >> $GITHUB_ENV
           echo "pr_from_sha=$(jq --raw-output .pull_request.head.sha $GITHUB_EVENT_PATH)" >> $GITHUB_ENV
           echo "pr_from_clone_url=$(jq --raw-output .pull_request.head.repo.clone_url $GITHUB_EVENT_PATH)" >> $GITHUB_ENV
           echo "pr_to_clone_url=$(jq --raw-output .pull_request.base.repo.clone_url $GITHUB_EVENT_PATH)" >> $GITHUB_ENV
           echo "pr_title=$(jq --raw-output .pull_request.title $GITHUB_EVENT_PATH)" >> $GITHUB_ENV
           echo "pr_number=$(jq --raw-output .pull_request.number $GITHUB_EVENT_PATH)" >> $GITHUB_ENV
+          echo "pr_owner=$(jq --raw-output .pull_request.head.user.login $GITHUB_EVENT_PATH)" >> $GITHUB_ENV
+          echo "commit_details=Null" >> $GITHUB_ENV
```

Comment:
What do you think about rename this to `description` and for a pull request use `.pull_request.body` as its contents?

Reply:
I thought about it but was worried about the PRs that have really long description and would it cause any regression if we index such huge text. 
But, given we are only indexing the data for failed tests, the volume will not be that huge so I guess I can add it.

Reply:
> regression if we index such huge text

I grabbed the description of one of the resent PRs [1] which was 1,902 bytes in size. If there are 13,000 documents one for each pull request that include this text in an OpenSearch index, it would use ~24 MB of space.  If every merged request had 10 test failures, that is 240 MB of data for _all_ pull requests in OpenSearch.

I don't think these test failures are very useful in a historical context of more than 6 months making this number much smaller.  I'd rather include too much data when reporting that can be trimmed back, then be too conservative and not be able to answer useful questions.

- [1] https://github.com/opensearch-project/OpenSearch/pull/12959